main.nf

```diff
@@ -683,6 +759,78 @@ process align_dia_runs {
 }
 
 
+/*
+ * STEP 11 - Reformat output for MSstats
+ */
+process prepare_for_msstats {
+   publishDir "${params.outdir}/"
+
+   input:
+    set val(id), val(Sample), val(Condition), file(dialignr_file) from DIALignR_result
+    file exp_design from input_exp_design.first()
+    set val(id), val(Sample), file(lib_file) from input_lib_used_I.first()
+
+   output:
+    set val(id), val(Sample), val(Condition), file("${Sample}_${Condition}_reformatted.csv") into msstats_file
+
+   when:
+    params.generate_plots & (params.pyprophet_global_fdr_level=='protein')
+
+   script:
+    """
+     TargetedFileConverter -in ${lib_file} \\
+                           -out ${lib_file.baseName}.tsv
+
+     reformat_output_for_msstats.py --input ${dialignr_file} --exp_design ${exp_design} --library ${lib_file.baseName}.tsv --output "${Sample}_${Condition}_reformatted.csv"
+    """
+}
+
+
+/*
+ * STEP 12 - Run MSstats
+ */
+process run_msstats {
+   publishDir "${params.outdir}/"
+
+   input:
+    set val(id), val(Sample), val(Condition), file(csv) from msstats_file.groupTuple(by:1)
+
+   output:
+    file "*.pdf"
```

Comment:
I think in these cases one should document what the output contains and how many of those files are generated (e.g., one per ms run? one in total?)

Reply:
done